src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -16,6 +18,44 @@ public class EdmVocabularyAnnotation : EdmElement, IEdmVocabularyAnnotation
         private readonly string qualifier;
         private readonly IEdmExpression value;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EdmVocabularyAnnotation"/> class without providing a value.
+        /// The given <see cref="IEdmTerm"/> should have the default value and can be translated to <see cref="IEdmExpression"/>.
+        /// </summary>
+        /// <param name="term">Term bound by the annotation.</param>
+        /// <param name="target">Element the annotation applies to.</param>
+        public EdmVocabularyAnnotation(IEdmTerm term, IEdmVocabularyAnnotatable target)
+            : this(term, target, null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EdmVocabularyAnnotation"/> class without providing a value.
+        /// The given <see cref="IEdmTerm"/> should have the default value and can be translated to <see cref="IEdmExpression"/>.
+        /// </summary>
+        /// <param name="term">Term bound by the annotation.</param>
+        /// <param name="target">Element the annotation applies to.</param>
+        /// <param name="qualifier">Qualifier used to discriminate between multiple bindings of the same property or type.</param>
+        /// <remarks>
+        /// The reason to use different order of parameters is because the constructors with three parameters are indistinguishable.

```

Comment:
This concerns me greatly and makes me wonder if a factory method that can have a name that more accurately reflects the behavior would be a better choice

Reply:
Maybe something like `public static EdmVocabularyAnnotation CreateFromDefaultTerm(...)`

Reply:
+1, I made changes. Please take a look. Thanks.

Reply:
Looks good, thanks!